js/main.js

```diff
@@ -0,0 +1,23 @@
+function randomNumber(min, max) {
+  if ((min < 0) || (max < 0) || (min === max)) {
+    return -1;
+  } else if (min > max) {
+    [min, max] = [max, min];
+  }
+  const r = Math.random() * (max - min + 1) + min;
+  return Math.floor(r);
+}
+const minNum = 0;
+const maxNum = 11;
+randomNumber(minNum, maxNum);
+
+
+function checkRange(inputtxt, maxlength) {
+  return (inputtxt.length <= maxlength) ? 1 :  -1;
```

Comment:
Тут по заданию нужно вернуть true или false. Достаточно вернуть вот так
```suggestion
  return inputtxt.length <= maxlength;
```